pkg/scheduler/apis/config/v1alpha1/types.go

```diff
@@ -50,27 +51,55 @@ type CandidateDeterminationStrategy string
 // SchedulerConfiguration provides the configuration for the SeedManager admission plugin.
 type SchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
-
-	Strategy CandidateDeterminationStrategy `json:"candidateDeterminationStrategy"`
 	// ClientConnection specifies the kubeconfig file and client connection
 	// settings for the proxy server to use when communicating with the apiserver.
-	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
+	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection,omitempty"`
 	// LeaderElection defines the configuration of leader election client.
-	LeaderElection LeaderElectionConfiguration `json:"leaderElection"`
+	LeaderElection LeaderElectionConfiguration `json:"leaderElection,omitempty"`
 	// Discovery defines the configuration of the discovery client.
-	Discovery DiscoveryConfiguration `json:"discovery"`
+	Discovery DiscoveryConfiguration `json:"discovery,omitempty"`
 	// LogLevel is the level/severity for the logs. Must be one of [info,debug,error].
-	LogLevel string `json:"logLevel"`
+	LogLevel string `json:"logLevel,omitempty"`
 	// Server defines the configuration of the HTTP server.
-	Server ServerConfiguration `json:"server"`
-	// RetrySyncPeriod is the duration how fast Shoots with an errornous operation are
+	Server ServerConfiguration `json:"server,omitempty"`
+	// Scheduler defines the configuration of the schedulers.
+	Schedulers SchedulerControllerConfiguration `json:"schedulers"`
```

Comment:
I don't see https://github.com/gardener/gardener/blob/master/example/20-componentconfig-gardener-scheduler.yaml and https://github.com/gardener/gardener/tree/master/charts/gardener adapted for this API change